src/textwatcher.js

```diff
@@ -43,7 +43,7 @@ export default class TextWatcher {
 		const model = this.model;
```

Comment:
Basically I think that adding an observable `isEnabled` property to this class will make easier to start/stop listening.

Reply:
👍